app/javascript/packages/address-search/index-spec.tsx

```diff
@@ -21,14 +21,17 @@ const DEFAULT_RESPONSE = [
   },
 ];
 
+const LOCATIONS_DUMMY_URL = 'https://login.gov/api/locations';
+const ADDRESSES_DUMMY_URL = 'https://login.gov/api/addresses';
```

Comment:
I wouldn't call these "DUMMY"? I thnik it's just `LOCATIONS_URL` and `ADDRESS_URL`?